lib/v2/nber/news.js

```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
```

Comment:
Please combine this file with `lib/v2/nber/papers.js`
You can use `ctx.path` to know the request path